### ISSUE TITLE
Accessibility Audits & Upgrades in Samvera

### ISSUE THREAD

**Issue by richardGreen** [projecthydra-labs/hydra-connect#262]
Kevin Kochanski

kevin@notch8.com

Federal law requires a standard of accessibility on any project that receives federal funds. Creating accessible applications is also socially responsible, and implementing these practices can help us set better expectations as community leaders. In a community dedicated to preservation and providing access to to a broad range of assets, accessibility standards should be a priority. We'll explain audits and certification levels, as well as accessibility features that could provide significant value to digital repositories. Using Hyku as an example, a developer will demonstrate practices that are easy to incorporate into the dev process, as well as demo practical examples.

**Comment by richardGreen** [comment]
8/22 Lea Ann Bradford withdrawn

**Comment by richardGreen** [comment]
Confirmed 8/23

**Comment by richardGreen** [comment]
8/27 Registered
